app/templates/publications/publication/case.hbs

```diff
@@ -3,23 +3,34 @@
 <div class="auk-u-m-4">
   <div class="auk-u-mb-4">
     <Publications::Publication::Case::Info::PublicationCaseInfoPanel
-      @publicationFlow={{this.model}} />
+      @publicationFlow={{this.model}}
+      class="auk-u-mb-3"
+    />
 
     <Publications::Publication::Case::Remark::RemarkPanel
-      @publicationFlow={{this.model}} />
+      @publicationFlow={{this.model}}
+      class="auk-u-mb-3"
+    />
 
-    <Publications::Publication::Case::Inscription::InscriptionPanel
-      @publicationFlow={{this.model}} />
+    <div class="auk-u-mb-3">
```

Comment:
Any specific reason you decided to add a wrapping `div` here instead of passing in the `auk-u-mb-3` class to the component via `...attributes`? You can add the `...attributes` to both the branches inside the component (so once in the normal panel, once in the `isEditing` panel.

Reply:
It seems like I forget to correctly refactor that instance. Thanks for noticing!